faunadb/query.py

```diff
@@ -385,6 +343,21 @@ def not_expr(boolean):
 
 #endregion
 
+#region Helpers
+
+@contextmanager
+def _auto_vars(n_vars):
+  if not hasattr(_thread_local, "fauna_lambda_var_number"):
+    _thread_local.fauna_lambda_var_number = 0
+  low_var_number = _thread_local.fauna_lambda_var_number
```

Comment:
The code here is a bit visually dense. Can you break it up a bit (eg line breaks, shorter names, etc)?